packages/ui-extensions-react/package.json

```diff
@@ -0,0 +1,43 @@
+{
+    "name": "@datadog/ui-extensions-react",
+    "version": "0.25.0",
+    "description": "Datadog UI Extensions - React helpers",
+    "homepage": "https://github.com/DataDog/apps/tree/master/packages/ui-extensions-react",
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/DataDog/apps.git",
+        "directory": "packages/ui-extensions-react"
+    },
+    "main": "dist/ui-extensions-react.min.js",
+    "types": "dist/index.d.ts",
+    "scripts": {
+        "build": "webpack --mode production",
+        "prepare": "yarn build",
+        "test": "yarn test:unit",
+        "test:unit": "jest",
+        "watch": "webpack --watch --mode development"
+    },
+    "files": [
+        "dist"
+    ],
+    "license": "Apache-2.0",
+    "publishConfig": {
+        "access": "public"
+    },
+    "devDependencies": {
+        "@testing-library/react-hooks": "7.0.2",
+        "@types/jest": "^26.0.23",
+        "clean-webpack-plugin": "^4.0.0-alpha.0",
+        "jest": "26.6.0",
+        "react-test-renderer": "17.0.2",
+        "ts-jest": "^26.5.5",
+        "ts-loader": "^9.1.1",
+        "webpack": "^5.35.1",
+        "webpack-cli": "^4.6.0",
+        "webpack-dev-server": "3.11.1"
+    },
+    "peerDependencies": {
+        "@datadog/ui-extensions-sdk": "0.25.0",
+        "react": "^17.0.2"
```

Comment:
These are peer dependencies because we depend on their APIs for this package's own APIs.